src/main/java/uk/ac/ebi/eva/pipeline/io/readers/NonAnnotatedVariantsMongoReader.java

```diff
@@ -36,7 +36,7 @@ public NonAnnotatedVariantsMongoReader(String databaseName, String collectionsVa
                                            MongoConnection connection) throws UnknownHostException {
         super();
 
-        setMongo(MongoDBHelper.getMongoClient(connection));
+        setMongo(new MongoDBHelper().getMongoClient(connection));
```

Comment:
We can't make it static because it requires the injection of the mapping converter, which is the reason for this refactoring (read the ticket description damn you! :stuck_out_tongue: ).

This was my mind slipping though, this class should look like [GeneWriterConfiguration](https://github.com/EBIvariation/eva-pipeline/pull/68/files/d2ba53ebe1fd80ec00afe3ca60a985fb72f03dbc#diff-d4abd9d30e525fc4f6584eaab081082e) does now.